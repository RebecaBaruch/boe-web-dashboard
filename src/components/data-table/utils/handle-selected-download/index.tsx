import { pdf } from '@react-pdf/renderer';
import {
  PDFDocumentGenerator,
  PDFDocumentProps,
} from '../../../../../src/utils/pdf-document-generator';

const downloadPDF = async (
  record: PDFDocumentProps,
  fileName: string = 'diagnosis_report.pdf',
) => {
  const pdfDocument = PDFDocumentGenerator(record) as React.ReactElement;

  const blob = await pdf(pdfDocument).toBlob();

  const link = document.createElement('a');
  link.href = URL.createObjectURL(blob);
  link.download = fileName;
  link.click();
};

export default downloadPDF;
