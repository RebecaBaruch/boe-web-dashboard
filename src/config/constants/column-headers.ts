import { Column } from 'components/data-table/types';

export const accountsColumns: Column[] = [
  { header: 'Id', accessor: 'id' },
  { header: 'Foto', accessor: 'picture' },
  { header: 'Nome', accessor: 'name' },
  { header: 'E-mail', accessor: 'email' },
  { header: 'N° de registros', accessor: 'registers' },
  { header: 'Data de ingresso', accessor: 'date' },
  { header: 'Ação', accessor: 'action' },
];
