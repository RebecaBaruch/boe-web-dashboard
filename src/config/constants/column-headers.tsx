import React from 'react';
import {
  Column,
  TableAnimalsData,
  TableDiagnosisData,
} from 'components/data-table/types';
import {
  AnimalIdContainer,
  AnimalPhoto,
  AuthorContainer,
  AuthorPhoto,
  ResultBadge,
  RiskBadge,
  RiskIndicator,
  StatusBadge,
} from 'components/diagnosis-components/diagnosis-components.components';

export const accountsColumns: Column[] = [
  { header: 'Id', accessor: 'id' },
  { header: 'Foto', accessor: 'picture' },
  { header: 'Nome', accessor: 'name' },
  { header: 'E-mail', accessor: 'email' },
  { header: 'N° de registros', accessor: 'registers' },
  { header: 'Data de ingresso', accessor: 'date' },
];

export const diagnosisListColumns: Column[] = [
  {
    header: '',
    accessor: 'riskLabel',
    render: (item: TableDiagnosisData): React.ReactNode => (
      <RiskIndicator risk={item.risk} />
    ),
  },
  {
    header: 'Id do animal',
    accessor: 'animalId',
    render: (item: TableDiagnosisData) => (
      <AnimalIdContainer>
        <AnimalPhoto src={item.animalPhoto} alt="Animal" />
        <span>{item.animalId}</span>
      </AnimalIdContainer>
    ),
  },
  {
    header: 'Resultado',
    accessor: 'result',
    render: (item: TableDiagnosisData) => (
      <ResultBadge result={item.result}>{item.result}</ResultBadge>
    ),
  },
  {
    header: 'Risco (em %)',
    accessor: 'risk',
    render: (item: TableDiagnosisData) => (
      <RiskBadge risk={item.risk}>{item.risk}%</RiskBadge>
    ),
  },
  {
    header: 'Autor da análise',
    accessor: 'author',
    render: (item: TableDiagnosisData) => (
      <AuthorContainer>
        <AuthorPhoto src={item.authorPhoto} alt="Author" />
        <span>{item.authorName}</span>
      </AuthorContainer>
    ),
  },
  {
    header: 'Data',
    accessor: 'date',
    render: (item: TableDiagnosisData) => <span>{item.date}</span>,
  },
];

export const animalsListColumns: Column[] = [
  {
    header: '',
    accessor: 'riskLabel',
    render: (item: TableAnimalsData): React.ReactNode => (
      <RiskIndicator risk={item.risk} />
    ),
  },
  {
    header: 'Id do animal',
    accessor: 'animalId',
    render: (item: TableAnimalsData) => (
      <AnimalIdContainer>
        <AnimalPhoto src={item.animalPhoto} alt="Animal" />
        <span>{item.animalId}</span>
      </AnimalIdContainer>
    ),
  },
  {
    header: 'Tipo de infecção',
    accessor: 'result',
    render: (item: TableAnimalsData) => (
      <ResultBadge result={item.result}>{item.result}</ResultBadge>
    ),
  },
  {
    header: 'Risco (em %)',
    accessor: 'risk',
    render: (item: TableAnimalsData) => (
      <RiskBadge risk={item.risk}>{item.risk}%</RiskBadge>
    ),
  },
  {
    header: 'Data',
    accessor: 'date',
    render: (item: TableAnimalsData) => <span>{item.date}</span>,
  },
  {
    header: 'Status de tratamento',
    accessor: 'status',
    render: (item: TableAnimalsData) => (
      <StatusBadge status={item.status ?? 'Indefinido'}>
        {item.status ?? 'Indefinido'}
      </StatusBadge>
    ),
  },
];
