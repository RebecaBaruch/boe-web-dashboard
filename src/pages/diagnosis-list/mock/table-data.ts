import {
  TableAnimalsData,
  TableDiagnosisData,
} from 'components/data-table/types';

export const diagnosisMockData: TableDiagnosisData[] = Array.from(
  { length: 20 },
  (_, index) => ({
    id: `AU${String(1000 + index).padStart(4, '0')}`, // Altere para 'id' ao invés de 'animalId'
    riskLabel: ['Alto', 'Médio', 'Baixo'][Math.floor(Math.random() * 3)],
    animalId: `AU${String(1000 + index).padStart(4, '0')}`, // Deixe o 'animalId' também para manter consistência
    animalPhoto: `https://placekitten.com/50/50?image=${index}`,
    authorName: ['Nelson Oliveira', 'Fabrício Neves'][index % 2],
    authorPhoto: `https://randomuser.me/api/portraits/men/${index}.jpg`,
    date: new Date(2024, 8, 17 + (index % 7)).toLocaleDateString('pt-BR'),
    diagnosisDate: new Date(2024, 8, 17 + (index % 7)).toLocaleDateString('pt-BR'),
    result: ['Berne', 'Dermatite', 'Dermatofitose', 'Dermatofilose'][
      Math.floor(Math.random() * 4)
    ],
    risk: Math.floor(Math.random() * 101),
  }),
);

export const animalsMockData: TableAnimalsData[] = Array.from(
  { length: 10 },
  (_, index) => ({
    id: `AU${String(1000 + index).padStart(4, '0')}`, // Altere para 'id' ao invés de 'animalId'
    riskLabel: ['Alto', 'Médio', 'Baixo'][Math.floor(Math.random() * 3)],
    animalId: `AU${String(1000 + index).padStart(4, '0')}`, // Deixe o 'animalId' também para manter consistência
    animalPhoto: `https://placekitten.com/50/50?image=${index}`,
    date: new Date(2024, 8, 17 + (index % 7)).toLocaleDateString('pt-BR'),
    result: ['Berne', 'Dermatite', 'Dermatofitose', 'Dermatofilose'][
      Math.floor(Math.random() * 4)
    ],
    risk: Math.floor(Math.random() * 101),
    status: ['Em andamento', 'Concluído', 'Sem tratamento'][
      Math.floor(Math.random() * 3)
    ],
  }),
);
