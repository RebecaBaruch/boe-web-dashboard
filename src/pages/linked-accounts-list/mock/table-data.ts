export const dataMock = [
  {
    id: '1',
    picture: 'link_da_foto_1',
    name: 'Claudio Buarque',
    email: 'claudio.buarque@gmail.com',
    registers: '02',
    date: '10/01/2024',
    action: 'Ação 1',
  },
  {
    id: '2',
    picture: 'link_da_foto_2',
    name: 'Fabrício Neves',
    email: 'fabricio.neves@gmail.com',
    registers: '12',
    date: '10/01/2024',
    action: 'Ação 2',
  },
  {
    id: '3',
    picture: 'link_da_foto_3',
    name: 'José Henrique',
    email: 'jose.henrique@gmail.com',
    registers: '03',
    date: '02/02/2024',
    action: 'Ação 3',
  },
  {
    id: '4',
    picture: 'link_da_foto_4',
    name: 'Luana Buarque',
    email: 'luana.buarque@gmail.com',
    registers: '02',
    date: '05/03/2024',
    action: 'Ação 4',
  },
  {
    id: '5',
    picture: 'link_da_foto_5',
    name: 'Augusto Buarque',
    email: 'augusto.buarque@gmail.com',
    registers: '04',
    date: '02/02/2024',
    action: 'Ação 5',
  },
  {
    id: '6',
    picture: 'link_da_foto_6',
    name: 'Nelson Oliveira',
    email: 'nelson.oliveira@gmail.com',
    registers: '07',
    date: '10/01/2024',
    action: 'Ação 6',
  },
  {
    id: '7',
    picture: 'link_da_foto_7',
    name: 'Mariana Silva',
    email: 'mariana.silva@gmail.com',
    registers: '01',
    date: '11/01/2024',
    action: 'Ação 7',
  },
  {
    id: '8',
    picture: 'link_da_foto_8',
    name: 'Roberto Carlos',
    email: 'roberto.carlos@gmail.com',
    registers: '05',
    date: '15/01/2024',
    action: 'Ação 8',
  },
  {
    id: '9',
    picture: 'link_da_foto_9',
    name: 'Tatiane Lopes',
    email: 'tatiane.lopes@gmail.com',
    registers: '06',
    date: '20/01/2024',
    action: 'Ação 9',
  },
  {
    id: '10',
    picture: 'link_da_foto_10',
    name: 'Ricardo Mendes',
    email: 'ricardo.mendes@gmail.com',
    registers: '10',
    date: '25/01/2024',
    action: 'Ação 10',
  },
  {
    id: '11',
    picture: 'link_da_foto_11',
    name: 'Aline Pires',
    email: 'aline.pires@gmail.com',
    registers: '03',
    date: '30/01/2024',
    action: 'Ação 11',
  },
  {
    id: '12',
    picture: 'link_da_foto_12',
    name: 'Samuel Gomes',
    email: 'samuel.gomes@gmail.com',
    registers: '04',
    date: '02/02/2024',
    action: 'Ação 12',
  },
  {
    id: '13',
    picture: 'link_da_foto_13',
    name: 'Juliana Santos',
    email: 'juliana.santos@gmail.com',
    registers: '09',
    date: '05/02/2024',
    action: 'Ação 13',
  },
  {
    id: '14',
    picture: 'link_da_foto_14',
    name: 'Fernanda Lima',
    email: 'fernanda.lima@gmail.com',
    registers: '11',
    date: '10/02/2024',
    action: 'Ação 14',
  },
  {
    id: '15',
    picture: 'link_da_foto_15',
    name: 'Gustavo Rocha',
    email: 'gustavo.rocha@gmail.com',
    registers: '08',
    date: '15/02/2024',
    action: 'Ação 15',
  },
  {
    id: '16',
    picture: 'link_da_foto_16',
    name: 'Bruna Almeida',
    email: 'bruna.almeida@gmail.com',
    registers: '02',
    date: '20/02/2024',
    action: 'Ação 16',
  },
  {
    id: '17',
    picture: 'link_da_foto_17',
    name: 'Lucas Ferreira',
    email: 'lucas.ferreira@gmail.com',
    registers: '05',
    date: '25/02/2024',
    action: 'Ação 17',
  },
  {
    id: '18',
    picture: 'link_da_foto_18',
    name: 'Larissa Costa',
    email: 'larissa.costa@gmail.com',
    registers: '06',
    date: '01/03/2024',
    action: 'Ação 18',
  },
  {
    id: '19',
    picture: 'link_da_foto_19',
    name: 'Vitor Hugo',
    email: 'vitor.hugo@gmail.com',
    registers: '12',
    date: '05/03/2024',
    action: 'Ação 19',
  },
  {
    id: '20',
    picture: 'link_da_foto_20',
    name: 'Sofia Martins',
    email: 'sofia.martins@gmail.com',
    registers: '07',
    date: '10/03/2024',
    action: 'Ação 20',
  },
];
