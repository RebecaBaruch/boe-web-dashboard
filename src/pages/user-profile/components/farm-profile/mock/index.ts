export const mockFarmData = {
  propertyName: 'Fazenda São João',
  farmPhone: '(13) 12345-6789',
  cep: '11900-000',
  cnpj: '12.345.678/0001-99',
  location: 'Itaquá, SP',
  address: 'Rua das Palmeiras, 123, Bairro Verde',
};
