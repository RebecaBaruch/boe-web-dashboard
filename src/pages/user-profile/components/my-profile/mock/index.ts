export const mockMyProfileData = {
  firstName: 'Claudio',
  lastName: 'Buarque',
  email: 'claudio.buarque@gmail.com',
  phone: '(11) 12345-6789',
  location: 'Registro, SP, Brasil',
};
