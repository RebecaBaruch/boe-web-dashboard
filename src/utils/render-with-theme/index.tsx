import React from 'react';
import { ThemeContextProvider } from '@telefonica/mistica';
import { render, RenderResult } from '@testing-library/react';
import { theme } from '../../config/theme';

export const renderWithTheme = (children: React.ReactNode): RenderResult => {
  return render(
    <ThemeContextProvider theme={theme}>{children}</ThemeContextProvider>,
  );
};
