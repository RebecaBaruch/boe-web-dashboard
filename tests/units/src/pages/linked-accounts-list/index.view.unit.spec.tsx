import React from 'react';
import { screen, within } from '@testing-library/react';
import LinkedAccountsList from '../../../../../src/pages/linked-accounts-list/view/linked-accounts-list.view';
import { renderWithTheme } from '../../../../../src/utils/render-with-theme';
import { dataMock } from './mock/table-data';
import { accountsColumns } from '../../../../../src/config/constants/column-headers';
describe('LinkedAccountsList', () => {
  it('should render LinkedAccountsList with text LinkedAccountsList', () => {
    const { container } = renderWithTheme(
      <LinkedAccountsList
        accountsData={dataMock}
        columnsHeader={accountsColumns}
      />,
    );

    expect(screen.getByText('Contas vinculadas')).toBeInTheDocument();

    const row = screen.getByText('Claudio Buarque').closest('tr');
    expect(row).toBeInTheDocument();
    if (row) {
      expect(
        within(row).getByText('claudio.buarque@gmail.com'),
      ).toBeInTheDocument();
      expect(within(row).getByText('02')).toBeInTheDocument();
      expect(within(row).getByText('10/01/2024')).toBeInTheDocument();
    }

    expect(container).toMatchSnapshot();
  });
});
