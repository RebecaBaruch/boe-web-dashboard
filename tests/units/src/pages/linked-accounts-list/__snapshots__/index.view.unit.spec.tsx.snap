// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkedAccountsList should render LinkedAccountsList with text LinkedAccountsList 1`] = `
<div
  style="isolation: isolate;"
>
  <div
    class="sc-dc1b69e8-0 bmwvjx"
  >
    <div
      data-component-name="Title3"
    >
      <h3
        class="splu5g7 splu5g5"
        data-component-name="Text"
        style="--splu5g0: 1.250rem; --splu5g2: 1.500rem; --splu5g1: 1.750rem; --splu5g3: 2.000rem; hyphens: auto; font-weight: 300; text-transform: inherit; text-decoration: inherit; overflow-wrap: anywhere; white-space: pre-line;"
      >
        <b>
          Contas vinculadas
        </b>
      </h3>
    </div>
    <table
      class="sc-73d4add1-0 cMwyLF"
    >
      <thead>
        <tr
          class="sc-73d4add1-1 gsFmqs"
        >
          <th
            class="sc-73d4add1-2 gkAQVs"
          >
            Id
          </th>
          <th
            class="sc-73d4add1-2 gkAQVs"
          >
            Foto
          </th>
          <th
            class="sc-73d4add1-2 gkAQVs"
          >
            Nome
          </th>
          <th
            class="sc-73d4add1-2 gkAQVs"
          >
            E-mail
          </th>
          <th
            class="sc-73d4add1-2 gkAQVs"
          >
            N° de registros
          </th>
          <th
            class="sc-73d4add1-2 gkAQVs"
          >
            Data de ingresso
          </th>
          <th
            class="sc-73d4add1-2 gkAQVs"
          >
            Ação
          </th>
        </tr>
      </thead>
      <tbody>
        <tr
          class="sc-73d4add1-1 gsFmqs"
        >
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            01
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            link_da_foto_1
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Claudio Buarque
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            claudio.buarque@gmail.com
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            02
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            10/01/2024
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Ação 1
          </td>
        </tr>
        <tr
          class="sc-73d4add1-1 gsFmqs"
        >
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            02
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            link_da_foto_2
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Fabrício Neves
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            fabricio.neves@gmail.com
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            12
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            10/01/2024
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Ação 2
          </td>
        </tr>
        <tr
          class="sc-73d4add1-1 gsFmqs"
        >
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            03
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            link_da_foto_3
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            José Henrique
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            jose.henrique@gmail.com
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            03
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            02/02/2024
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Ação 3
          </td>
        </tr>
        <tr
          class="sc-73d4add1-1 gsFmqs"
        >
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            04
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            link_da_foto_4
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Luana Buarque
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            luana.buarque@gmail.com
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            02
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            05/03/2024
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Ação 4
          </td>
        </tr>
        <tr
          class="sc-73d4add1-1 gsFmqs"
        >
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            05
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            link_da_foto_5
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Augusto Buarque
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            augusto.buarque@gmail.com
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            04
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            02/02/2024
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Ação 5
          </td>
        </tr>
        <tr
          class="sc-73d4add1-1 gsFmqs"
        >
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            06
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            link_da_foto_6
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Nelson Oliveira
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            nelson.oliveira@gmail.com
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            07
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            10/01/2024
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Ação 6
          </td>
        </tr>
        <tr
          class="sc-73d4add1-1 gsFmqs"
        >
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            07
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            link_da_foto_7
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Mariana Silva
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            mariana.silva@gmail.com
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            01
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            11/01/2024
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Ação 7
          </td>
        </tr>
        <tr
          class="sc-73d4add1-1 gsFmqs"
        >
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            08
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            link_da_foto_8
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Roberto Carlos
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            roberto.carlos@gmail.com
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            05
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            15/01/2024
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Ação 8
          </td>
        </tr>
        <tr
          class="sc-73d4add1-1 gsFmqs"
        >
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            09
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            link_da_foto_9
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Tatiane Lopes
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            tatiane.lopes@gmail.com
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            06
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            20/01/2024
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Ação 9
          </td>
        </tr>
        <tr
          class="sc-73d4add1-1 gsFmqs"
        >
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            10
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            link_da_foto_10
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Ricardo Mendes
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            ricardo.mendes@gmail.com
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            10
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            25/01/2024
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Ação 10
          </td>
        </tr>
        <tr
          class="sc-73d4add1-1 gsFmqs"
        >
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            11
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            link_da_foto_11
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Aline Pires
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            aline.pires@gmail.com
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            03
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            30/01/2024
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Ação 11
          </td>
        </tr>
        <tr
          class="sc-73d4add1-1 gsFmqs"
        >
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            12
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            link_da_foto_12
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Samuel Gomes
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            samuel.gomes@gmail.com
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            04
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            02/02/2024
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Ação 12
          </td>
        </tr>
        <tr
          class="sc-73d4add1-1 gsFmqs"
        >
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            13
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            link_da_foto_13
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Juliana Santos
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            juliana.santos@gmail.com
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            09
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            05/02/2024
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Ação 13
          </td>
        </tr>
        <tr
          class="sc-73d4add1-1 gsFmqs"
        >
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            14
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            link_da_foto_14
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Fernanda Lima
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            fernanda.lima@gmail.com
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            11
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            10/02/2024
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Ação 14
          </td>
        </tr>
        <tr
          class="sc-73d4add1-1 gsFmqs"
        >
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            15
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            link_da_foto_15
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Gustavo Rocha
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            gustavo.rocha@gmail.com
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            08
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            15/02/2024
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Ação 15
          </td>
        </tr>
        <tr
          class="sc-73d4add1-1 gsFmqs"
        >
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            16
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            link_da_foto_16
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Bruna Almeida
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            bruna.almeida@gmail.com
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            02
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            20/02/2024
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Ação 16
          </td>
        </tr>
        <tr
          class="sc-73d4add1-1 gsFmqs"
        >
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            17
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            link_da_foto_17
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Lucas Ferreira
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            lucas.ferreira@gmail.com
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            05
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            25/02/2024
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Ação 17
          </td>
        </tr>
        <tr
          class="sc-73d4add1-1 gsFmqs"
        >
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            18
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            link_da_foto_18
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Larissa Costa
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            larissa.costa@gmail.com
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            06
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            01/03/2024
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Ação 18
          </td>
        </tr>
        <tr
          class="sc-73d4add1-1 gsFmqs"
        >
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            19
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            link_da_foto_19
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Vitor Hugo
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            vitor.hugo@gmail.com
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            12
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            05/03/2024
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Ação 19
          </td>
        </tr>
        <tr
          class="sc-73d4add1-1 gsFmqs"
        >
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            20
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            link_da_foto_20
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Sofia Martins
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            sofia.martins@gmail.com
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            07
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            10/03/2024
          </td>
          <td
            class="sc-73d4add1-3 gOsREM"
          >
            Ação 20
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;
